### PR TITLE
Updates the reminders component to match updated designs

### PR DESCRIPTION
**Pull request by tomkingston**
**Marked as DO NOT MERGE until design is checked by Anna**

## Description of change

Updates the reminders component to match the [updated designs](https://glc39r.axshare.com/#id=ozm45m&p=spec_reminders&g=1).

## Test instructions

With the dashboard feature flag enabled, the top left should show a reminders component containing 3 outstanding propositions. The proposition name should link to the list of propositions for the project.

### Before

Screenshot from 2021-04-01 12-45-26

### After

Screenshot from 2021-04-06 09-22-22

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/client/components/InvestmentReminders/OutstandingPropositions.jsx [hunk 1]
```diff
@@ -57,16 +61,18 @@ const OutstandingPropositions = ({ results, count }) => {
...
             <StyledProjectLink
-              href={urls.investments.projects.proposition(
-                investment_project.id,
-                id
+              href={urls.investments.projects.propositions(
+                investment_project.id
               )}
             >
-              {investment_project.project_code}
+              {name}
```
Comment (ccp92) [inline src/client/components/InvestmentReminders/OutstandingPropositions.jsx:71]: Looking at the investment_project object, it looks like name is already part of it?
Do you need to pass in name separately to this map or could you not just call `investment_project.name` from the existing argument?

**Review by ccp92 — APPROVED** [review]
One minor comment but otherwise lgtm

File: src/client/components/InvestmentReminders/OutstandingPropositions.jsx (hunk 1 above)
Comment (tomkingston) [reply src/client/components/InvestmentReminders/OutstandingPropositions.jsx:71]: Hi Chris, so `investment_project.name` is the name of the investment project whereas `name` is the name of the proposition, which are two different things - the new design wants us to show the name of the proposition here.

**Review by paulgain — APPROVED** [review]
Nice :-)

**Review by ian-leggett — APPROVED** [review]

**Pull request merged**